src/rhub/scheduler/jobs.py

```diff
@@ -89,15 +91,15 @@ def delete_expired_clusters(params):
     """
     now = date_now()
 
-    expired_clusters = rhub.lab.model.Cluster.query.filter(
+    expired_clusters = lab_model.Cluster.query.filter(
```

Comment:
We handle expired 'reservations' differently than we handle expired 'lifespans'.  Expired lifespans can just be marked to be deleted.  However with reservations, we allow a 'grace' period.  On each of the 1st, 2nd and 3rd days past the reservation expiration, we send a notification (email) to the owner of the cluster to let them know that their cluster is expired.  We request that they either extend the reservation or delete the cluster if it is no longer needed.  If by the 4th day, the owner has not taken any action, we are cleared to delete the cluster.